tests/test-webapps/test-owb-cdi-webapp/src/main/webapp/WEB-INF/jetty-env.xml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
 
 <Configure id="webAppCtx" class="org.eclipse.jetty.webapp.WebAppContext">
+  <Call class="java.lang.System" name="setProperty">
+    <Arg>openwebbeans.web.sci.active</Arg>
+    <Arg type="String">true</Arg>
```

Comment:
This is the critical piece missing to allow openwebbean's own CDI from functioning.